.circleci/config.yml

```
@@ -7,12 +7,23 @@ install_hex_rebar: &install_hex_rebar
       mix local.hex --force
       mix local.rebar --force
 
+install_system_deps: &install_system_deps
+  run:
+    name: Install system dependencies
+    command: |
+      apk add build-base linux-headers libmnl-dev libnl3-dev curl
```

```suggestion
      apk add build-base
```
I think this should be all that is needed in order to have `git`